deploy/crds/postgresql_v1alpha1_database_cr.yaml

```diff
@@ -7,3 +7,4 @@ spec:
   image: docker.io/postgres
   imageName: postgres
   dbName: postgres
+  bindingData: ['secret': 'something1', 'secret': 'something2', 'secret': 'something3']
```

Comment:
The CR is created by the 'developer' not the operator author - whereas the information on what is interesting to bind, is put in the CSV, by the operator author.

Reply:
Okay, so the changes should be in CRD, not CR, right?